source/extensions/filters/http/composite/action.cc

```diff
@@ -15,6 +15,38 @@ Matcher::ActionFactoryCb ExecuteFilterActionFactory::createActionFactoryCb(
       const envoy::extensions::filters::http::composite::v3::ExecuteFilterAction&>(
       config, validation_visitor);
 
+  if (composite_action.has_dynamic_config() && composite_action.has_typed_config()) {
+    throwEnvoyExceptionOrPanic(
```

Comment:
I believe if you main merge, we've stopped use OrPanic wrappers in this file.

Reply:
Done